src/components/TechGala/TechGalaBanner/TechGalaBanner.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function TechGalaBanner(props) {
+	return (
+        <div>
+            <img src={process.env.PUBLIC_URL + "/images/techgala/techgala-banner-light.jpg"} alt="2021 tech gala banner" />
```

Comment:
Would it be possible to use the dark-mode banner here? It's the final marketing we went with, though I do realize that it has time and date info that might not be entirely relevant since the event has already passed.
![image](https://user-images.githubusercontent.com/23728476/109917233-d1e83b00-7c69-11eb-8696-bdc508483cf9.png)


```suggestion
            <img src={process.env.PUBLIC_URL + "/images/techgala/techgala-banner-dark.jpg"} alt="2021 tech gala banner" />
```